### DISCUSSION TITLE
Anyone have a sprite_sheet that works with prefabs/sprite_sheet_animation.py?

### DISCUSSION THREAD

**Discussion by ghost** [pokepetter/ursina#708] [Q&A]
Cannot seem to find the asset for the sprite_sheet_animation.py example, which showcases SpriteSheetAnimation(). If anyone has an asset that works with this example, please let me know!

**Comment by ZombiesLoveZGames** [comment]
If you're still looking, this texture seems to do nicely, might need to adjust the file paths in order to properly display it, but it works!
![sprite_sheet](https://github.com/user-attachments/assets/3fd94f03-9740-4fba-8c23-9a04ae4baa0d)
